modules/openshift-cluster-maximums-environment.adoc

```diff
@@ -0,0 +1,95 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/planning-your-environment-according-to-object-maximums.adoc
+
+[id="cluster-maximums-environment_{context}"]
+= {product-title} environment and configuration on which the cluster maximums are tested
+
+Google Cloud Platform:
+
+[options="header",cols="8*"]
+|===
+| Node |Flavor |vCPU |RAM(GiB) |Disk type|Disk size(GiB)/IOPS |Count |Region
+
+| Master/Etcd
+| n1-highmem-16
+| 16
+| 104
+| Regional/Zonal SSD 
+| 220
+| 3
+| us-east4
+
+| Infra footnoteref:[infranodesgcp,Infra nodes are used to host openshift-monitoring, ingress and router components to make sure they have enough resources to run at large scale. Procedure on creating Infra machinesets can be found at https://docs.openshift.com/container-platform/4.2/machine_management/creating-infrastructure-machinesets.html.]
```

Comment:
Infra nodes are used to host openshift-monitoring, Ingress, and router components to ensure they have enough resources to run at a large scale.

Unfortunately, we have to remove the link to the procedure, as we are currently not allowed to include internal xrefs within modules. Perhaps we can move this to a new "Related Information" section of the "assembly" file (as seen in [this template](Related Information)), which in this case is the planning-your-environment-according-to-object-maximums.adoc.